cartographer_ros/cartographer_ros/map_builder_bridge.cc

```diff
@@ -24,12 +24,17 @@
 namespace cartographer_ros {
 
 constexpr double kTrajectoryLineStripMarkerScale = 0.07;
+constexpr double kTrajectoryColorManagerSaturation = 0.8;
+constexpr double kTrajectoryColorManagerValue = 0.75;
 
 MapBuilderBridge::MapBuilderBridge(const NodeOptions& node_options,
                                    tf2_ros::Buffer* const tf_buffer)
     : node_options_(node_options),
       map_builder_(node_options.map_builder_options),
-      tf_buffer_(tf_buffer) {}
+      tf_buffer_(tf_buffer),
+      trajectory_color_manager_(node_options.trajectory_hue_first,
+                                kTrajectoryColorManagerSaturation,
```

Comment:
It is best not to eagerly generalize code if it adds complexity. For example it is unclear whether there will ever be a need in this code base for having different saturation and value. So it is best to remove these parameters. If we need them we could still refactor the code then. If we do not, the current code is overly complicated for no good reason.

Similar with the hue option: If there is need, we can introduce it. But it seems doubtful to me at least.